src/libs/actions/Session/index.ts

```diff
@@ -331,6 +331,15 @@ function signInWithShortLivedAuthToken(email: string, authToken: string) {
                 isLoading: true,
             },
         },
+        // We are making a temporary modification to 'signedInWithShortLivedAuthToken' to ensure that 'App.openApp' will be called at least once
+        // Subsequently, we revert it back to the default value (undefined) in 'successData' to ensure the user is logged out on refresh
```

Comment:
I think that this comment could be moved to the place where this happens

Reply:
Sure I will move that